src/Device/deviceActions.js

```diff
@@ -289,19 +289,30 @@ export const selectAndSetupDevice = (
         const promiseChoice = (message, choices) => (
             getDeviceSetupUserInput(dispatch, message, choices)
         );
+        const allowCustomDevice = false;
 
         await releaseCurrentDevice();
 
-        setupDevice(device, { promiseConfirm, promiseChoice, ...deviceSetup })
+        setupDevice(
+            device,
+            {
+                promiseConfirm,
+                promiseChoice,
+                allowCustomDevice,
+                ...deviceSetup,
+            },
```

Comment:
Probably nicer is to extract this config object into a constant, in which you can then also inline `promiseConfirm` and `promiseChoice`. And if you even make `getDeviceSetupUserInput` curryable, then it looks just like this:

```js
const deviceSetupConfig = {
    promiseConfirm: getDeviceSetupUserInput(dispatch),
    promiseChoice: getDeviceSetupUserInput(dispatch),
    allowCustomDevice: false,
    ...deviceSetup,
};
setupDevice(device, deviceSetupConfig);
```

Reply:
Nice suggestion :+1: